fr/FAQ.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Objections et réponses
+ref: faqfr
```

Comment:
the `ref` field must be the same in every version of the same page. It is used by Jekyll to know that there are other translations of the same page.